docs/configs/site.js

```diff
@@ -24,7 +24,7 @@ export default {
   port: 8080,
   domain: 'dolphinscheduler.apache.org',
   copyToDist: ['asset', 'img', 'file', '.asf.yaml', 'sitemap.xml', '.nojekyll', '.htaccess', 'googled0df7b96f277a143.html'],
-  docsLatest: '3.0.1',
```

Comment:
This file should not change, we use change branch `dev` one instead of branch `3.0.2-prepare` one

Reply:
Got it, thx : )